compiler/natives/src/net/http/http.go

```diff
@@ -13,18 +13,29 @@ import (
 	"github.com/gopherjs/gopherjs/js"
 )
 
-var DefaultTransport RoundTripper = &XHRTransport{}
+var DefaultTransport = func() RoundTripper {
+	if fetchAPI, streamsAPI := js.Global.Get("fetch"), js.Global.Get("ReadableStream"); fetchAPI != js.Undefined && streamsAPI != js.Undefined {
```

Comment:
Ah, I overlooked that or didn't think of it. It looks simpler, so I'll do it.

I think I initially tried to pass the variable into the transport, e.g.:

```
if xhrAPI := js.Global.Get("XMLHttpRequest"); xhrAPI != js.Undefined {
	return &XHRTransport{xhrAPI}
}
```

But reverted because it was a bad idea. It meant that zero value was no longer valid, and there's no performance gain in storing `js.Global.Get("XMLHttpRequest")` in a variable.